src/hooks/useStore.js

```diff
@@ -1,52 +1,58 @@
 import create from 'zustand';
+//import {persist} from 'zustand/middleware';
 
-const useStore = create(set => ({
-	currentExercise: {
-		exercise: {minutes: 0, seconds: 0},
-		repetition: 1,
-		pause: {minutes: 0, seconds: 0},
-		sets: 1,
-		setPause: {minutes: 0, seconds: 0},
-	},
-	setExercise: newExercise => {
-		set(state => {
-			return {
-				currentExercise: {...state.currentExercise, exercise: newExercise},
-			};
-		});
-	},
-	setRepetition: newRepetitions => {
-		set(state => {
-			return {
-				currentExercise: {...state.currentExercise, repetition: newRepetitions},
-			};
-		});
-	},
-	setPause: newPause => {
-		set(state => {
-			return {
-				currentExercise: {...state.currentExercise, pause: newPause},
-			};
-		});
-	},
-	setSets: newSets => {
-		set(state => {
-			return {
-				currentExercise: {...state.currentExercise, sets: newSets},
-			};
-		});
-	},
-	setPauseSets: newPauseSets => {
-		set(state => {
-			return {
-				currentExercise: {...state.currentExercise, setPause: newPauseSets},
-			};
-		});
-	},
-	timerOn: true,
-	changeToTrue: () => set(state => ({timerOn: (state.timerOn = true)})),
-	changeToFalse: () => set(state => ({timerOn: (state.timerOn = false)})),
-}));
+const useStore = create(
+	//	persist(
+	set => ({
+		currentExercise: {
+			exercise: {minutes: 0, seconds: 0},
+			repetition: 1,
+			pause: {minutes: 0, seconds: 0},
+			sets: 1,
+			setPause: {minutes: 0, seconds: 0},
+		},
+		setExercise: newExercise => {
+			set(state => {
+				return {
+					currentExercise: {...state.currentExercise, exercise: newExercise},
+				};
+			});
+		},
+		setRepetition: newRepetitions => {
+			set(state => {
+				return {
+					currentExercise: {...state.currentExercise, repetition: newRepetitions},
+				};
+			});
+		},
+		setPause: newPause => {
+			set(state => {
+				return {
+					currentExercise: {...state.currentExercise, pause: newPause},
+				};
+			});
+		},
+		setSets: newSets => {
+			set(state => {
+				return {
+					currentExercise: {...state.currentExercise, sets: newSets},
+				};
+			});
+		},
+		setPauseSets: newPauseSets => {
+			set(state => {
+				return {
+					currentExercise: {...state.currentExercise, setPause: newPauseSets},
+				};
+			});
+		},
+		timerOn: false,
+		changeToTrue: () => set(state => ({timerOn: (state.timerOn = true)})),
+		changeToFalse: () => set(state => ({timerOn: (state.timerOn = false)})),
```

Comment:
Maybe you can make it simpler this way, so you don't need to function to change the state: 

```javascipt
timerOn: false,
  setTimerOn: () => set(state => { timerOn: !state.timerOn }),
```

Reply:
as we talked about it, its better as it is, but thanks a lot for the info i tried yours and it doesnt fit unfortunatley to my case, but you are in generall right